all-actors/src/main/java/org/sunbird/serviceimpl/CertsServiceImpl.java

```diff
@@ -181,7 +182,19 @@ public Response download(Request request) throws BaseException {
         try {
             Map<String, Object> certReqMap = new HashMap<>();
             Map<String,String>requestMap=new HashMap<>();
-            requestMap.put(JsonKeys.PDF_URL,(String)request.getRequest().get(JsonKeys.PDF_URL));
+
+            if(ActorOperations.DOWNLOADV2.getOperation().equals(request.getOperation())) {
+                String certUrl = (String) request.getRequest().get(JsonKeys.CERT_URL);
+                String pdfUrl = (String) request.getRequest().get(JsonKeys.PDF_URL);
+
+                if (StringUtils.isNotEmpty(certUrl)) {
+                    requestMap.put(JsonKeys.PDF_URL, certUrl); //TODO-Change to CERT_URL if required
```

Comment:
Lets fix this as well as cert_url. If we dont do, other developers in future will cry.
Perhaps, we can have same endpoint 
- accept both pdfUrl and certUrl 
- if both are passed it will use certUrl.